spec/services/create_subscriber_list_service_spec.rb

```diff
@@ -0,0 +1,132 @@
+RSpec.describe CreateSubscriberListService do
+  describe ".call" do
+    let(:user) { create :user }
+    let(:existing_list_query) { double(FindExactQuery, exact_match: nil) }
+
+    let(:params) do
+      ActionController::Parameters.new(
+        title: "This is a sample title",
```

Comment:
Should we be testing our permitted params expectations somewhere? Ensuring that we don't accidentally open a hole here

Reply:
Hmm, interesting. Could you expand a bit on what you mean by expectations? I initially thought you meant protecting against the code "dumbly" passing the entire params hash into the record. But I don't think that's a likely coding avenue, as it's quite hard to go down that route in practice.

Perhaps you're referring more to the params that _are_ permitted? So can we inject a nasty title, for example? I think we're generally protected against that by ActiveRecord and the API adapters escaping everything, everywhere. I'm still interested, though: is there a particular scenario you're concerned about?

Reply:
Yeah, it was more about the former. I think I was looking at the `signon_user_id` field and wondering whether that could get overridden.

Reply:
Thanks for clarifying. I do think it's unlikely, as this is an internal API. There doesn't seem to be a precedent for testing for bad behaviour, except for params that are sourced from user input in frontend apps.

Interestingly, the app doesn't use `signon_user_id` for anything. Perhaps the idea was to support auditing. I could add a test to prove that it's not overridable by params, but the value seems pretty low.

Do you think there's a missing test here?

Reply:
Possibly - looking back it does seem as though it was added everywhere to support something that might have been done in the future. Not in scope for this change though